context/context.go

```diff
@@ -19,7 +20,18 @@ import (
 )
 
 var (
-	clientGRPCPort string
+	clientGRPCPort     string
+	needWrapComponents = []string{
```

Comment:
needWrapComponents => bindingQueueComponents
and it shouldn't be an array, should be a map to speed up searching : map[string]bool